spec/lib/simp/beaker_helpers_spec.rb

```diff
@@ -80,11 +80,11 @@ def initialize
 
       # this logic won't work properly without code changes that just aren't worth it because
       # Puppet 4 is MD soon....
-      it "maps to appropriate Puppet version when '<' operator specified in version" do
-        pending 'fails because matches 4.x table'
-        allow(@helper).to receive(:`).with('gem search -ra -e puppet').and_return(gem_search_results)
-        expect( @helper.latest_puppet_agent_version_for('< 5.5') ).to match /5.4.0/
-      end
+      # it "maps to appropriate Puppet version when '<' operator specified in version" do
```

Comment:
Did you mean to leave this commented out?

Reply:
Yes, and the comment is still relevant. Setting it to pending is no longer sufficient, however. At least on my system